server/src/main/kotlin/suwayomi/tachidesk/graphql/mutations/ChapterMutation.kt

```diff
@@ -89,36 +91,40 @@ class ChapterMutation {
         }
     }
 
-    fun updateChapter(input: UpdateChapterInput): UpdateChapterPayload {
-        val (clientMutationId, id, patch) = input
+    fun updateChapter(input: UpdateChapterInput): DataFetcherResult<UpdateChapterPayload?> {
```

Comment:
Is making everything nullable changing the scheme?

Reply:
yes, the results will be nullable